spot_driver/launch/spot_driver.launch.py

```diff
@@ -2,51 +2,134 @@
 
 import launch
 import launch_ros
-import xacro
+import yaml
+from ament_index_python.packages import get_package_share_directory
 from launch import LaunchContext, LaunchDescription
 from launch.actions import DeclareLaunchArgument, OpaqueFunction
-from launch.substitutions import LaunchConfiguration
+from launch.conditions import IfCondition
+from launch.substitutions import (
+    Command,
+    FindExecutable,
+    LaunchConfiguration,
+    PathJoinSubstitution,
+)
 from launch_ros.substitutions import FindPackageShare
 
 
-def launch_robot_state_publisher(context: LaunchContext, has_arm: LaunchConfiguration, ld: LaunchDescription) -> None:
+def create_rviz_config(robot_name: str) -> None:
+    """Writes a configuration file for rviz to visualize a single spot robot"""
+    PACKAGE = "spot_driver"
+
+    RVIZ_TEMPLATE_FILENAME = os.path.join(get_package_share_directory(PACKAGE), "rviz", "spot_template.yaml")
+    RVIZ_OUTPUT_FILENAME = os.path.join(get_package_share_directory(PACKAGE), "rviz", "spot.rviz")
+
+    with open(RVIZ_TEMPLATE_FILENAME, "r") as template_file:
+        config = yaml.safe_load(template_file)
+        # Add robot models for each robot
+        for display in config["Visualization Manager"]["Displays"]:
+            if "RobotModel" in display["Class"]:
+                display["Description Topic"]["Value"] = f"/{robot_name}/robot_description"
```

Comment:
when `{robot_name}` is not provided, it errors looking for the model because the topic looks for `/robot_description`:
![image](https://github.com/bdaiinstitute/spot_ros2/assets/137220849/a94f6546-97a2-4590-a55b-0973c41d4f9a)